### PR TITLE
Add title

### PR DESCRIPTION
**Pull request by MarcalAmaral**

### CODE REVIEW

**Review by MarcalCampos — COMMENTED** [review]
Review add title #2

File: index.html [hunk 1]
```diff
@@ -1 +1,2 @@
 <h1>Hello World!</h1>
+<h1>Olá, Mundo!</h1>
```
Comment (MarcalCampos) [inline index.html]: Could you remove the `Olá` string and make the `Mundo` string red?

Comment (MarcalAmaral) [reply index.html]: Sure, can do!

**Review by MarcalCampos — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by MarcalCampos — APPROVED** [review]
Well done Marcal!

**Pull request merged**